web-registry/public/_redirects

```diff
@@ -1,5 +1,2 @@
+https://registry.regen.network/*  https://app.regen.network/:splat 301!
```

Comment:
this needs to be before the `/*   /index.html   200` redirect, otherwise it will never match.